.yarnrc.yml

```diff
@@ -1,6 +1,7 @@
 logFilters:
-  # react-native-web overrides react-native, so we don't need to have react-native installed
```

Comment:
Why remove this helpful comment?

Reply:
I'll put it back in. I didn't know it got lost.